kalite/distributed/static/js/distributed/inline/models.js

```diff
@@ -0,0 +1,9 @@
+// Model for the inline tutorial system that fetches from tastypie resource "NarrativeResource"
+
+var NarrativeModel = Backbone.Model.extend({
+    // urlRoot since we are using a model outside of a backbone collection,
+    // enables the url function to generate URLs based on the model id.
+    urlRoot: function() {
+        return '/api/inline/narrative';
```

Comment:
Okay, I just merged it. If you pull the latest `develop` here you can use the `window.Urls`, which has callable attributes corresponding to url names (which you can inspect by using the `show_urls` management command). Is that right @rtibbles?